regolith/helpers/a_presentationhelper.py

```diff
@@ -119,6 +126,61 @@ def construct_global_ctx(self):
         rc.coll = f"{TARGET_COLL}"
         if not rc.database:
             rc.database = rc.databases[0]["name"]
+
+        # if an expense database is specified using the --expense-db option,
+        if rc.expense_db:
+
+            # check that it's known / find it in rc.databases. Store its index number so we can access its other keys inside rc.databases
+            index = 0
+            db_known = False
+            while index < len(rc.databases):
+                if rc.databases[index]["name"] == rc.expense_db:
+                    db_known = True
+                    break
+                index += 1
+            expense_db_index = index
+
+            # If the database is not known, exit with descriptive error message
+            if not db_known:
+                 raise RuntimeError(
+                    f"WARNING: The expense database provided, {rc.expense_db}, does not exist. "
+                    "Please rerun specifying a known private database. "
+                    "(Consider checking your spelling against the known databases listed in regolithrc.json.)"
+                )
+
+            # if the specified expense database is public and the --force option wasn't passed, exit with a warning message
+            if (rc.databases[expense_db_index]["public"] == True) and (not rc.force):
+                raise RuntimeError(
+                    "WARNING: The expense database provided is not private. Please rerun specifying a known private database, "
```

Comment:
this is a good error message.  Here I am not altogether sure whether it should be WARNING or ERROR.  Strictly, since we are raising an exception it should be ERROR.  Another approach is to not raise an exception here but call `sys.exit` and leave it as a warning.   I think this may be more friendly to the user because it will look less like a crash.